code/game/objects/items/handcuffs.dm

```diff
@@ -537,6 +537,10 @@
 	gender = NEUTER
 	///Amount of time to knock the target down for once it's hit in deciseconds.
 	var/knockdown = 0
+	///Used for ensnaring the target when hit in after_throw override
+	var/is_mob_hit = FALSE
```

Comment:
Is this variable really necessary? You always use it in conjunction with ensnare_mob_ref anyways

Reply:
Yeah, now that I think about it... It is possible to just use weakref. Thanks for mentioning it.